src/test/app/AMM_test.cpp

```diff
@@ -6908,6 +6924,8 @@ struct AMM_test : public jtx::AMMTest
         testFixAMMOfferBlockedByLOB(all - fixAMMv1_1);
         testLPTokenBalance(all);
         testLPTokenBalance(all - fixAMMv1_1);
+        testIssuerClawback(all);
+        testIssuerClawback(all | featureAMMClawback);
```

Comment:
should be all - featureAMMClawback

Reply:
Since I use `REGISTER_FEATURE(AMMClawback, Supported::no, VoteBehavior::DefaultNo)` in Feature.cpp, featureAMMClawback is not in all features by default. Do you want to set featureAMMClawback Supported::yes by default?

Reply:
all amendments are enabled by default in test cases

Reply:
if tracing from AMM_test.cpp `FeatureBitset const all{jtx::supported_amendments()};` (https://github.com/yinyiqian1/rippled/pull/1/files#diff-2449dcf99b9cb0572a44031f6e2f9ddf8efa29fc4d96e657f86a80502a6039beL6868-R6884),  it only contains supported ones.

Reply:
could you try adding a breakpoint in the `if (!features[featureAMMClawback])` condition to see if it ever hits inside the if-statement.

Reply:
Yes. it will goes into if statement